Functions.Templates/Templates/HttpTrigger-CSharp-Isolated/.template.config/template.json

```diff
@@ -64,6 +64,17 @@
                 "projectFileExtensions": ".csproj"
             }
         },
+        {
+            "Description": "Adding Reference to Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore Nuget package",
```

Comment:
We only need this step for some apps (non-framework versions). @soninaren Any guidance for how to go about this?

Reply:
Can you elaborate on some apps part?

Reply:
This package shouldn't be added for net framework apps, but it seems like this template is shared for both netframework and netcore.